hepcrawl/spiders/pos_spider.py

```diff
@@ -32,24 +32,17 @@ class POSSpider(Spider):
     """POS/Sissa crawler.
 
     Extracts from metadata:
-        * title
-        * article-id
-        * conf-acronym
-        * authors
-        * affiliations
-        * publication-date
-        * publisher
-        * license
-        * language
-        * link
+        todo:: be added...
 
     Example:
         ::
 
-            $ scrapy crawl PoS -a source_file=file://`pwd`/tests/responses/pos/sample_pos_record.xml
+            $ scrapy crawl PoS -a source_file=file://`pwd`/tests/unit/responses/pos/
+            sample_pos_record.xml
```

Comment:
Here you should use something like:
```
    $ my huge \\
        big long \\
        command
```
So then it renders nice in the page too and can be copied and pasted to the console, otherwise you get a huge command that goes out of the screen.